.github/workflows/changelog-bot.yml

```diff
@@ -0,0 +1,19 @@
+name: Changelog Bot
+
+on:
+  push:
+    paths-ignore:
+      - CHANGELOG.md
+
+jobs:
+  changelog-bot:
+    runs-on: ubuntu-latest
+    name: Update CHANGELOG.md
+    steps:
+      - name: Update Changelog
+        uses: ponylang/changelog-bot-action@0.2.1
```

Comment:
Shouldn't this use version 0.3.0?

Reply:
No. the latest version of changelog-bot is 0.2.1